neuro_san/http_sidecar/handlers/connectivity_handler.py

```diff
@@ -23,15 +23,21 @@ class ConnectivityHandler(BaseRequestHandler):
     Handler class for neuro-san "connectivity" API call.
     """
 
-    async def get(self):
+    async def get(self, agent_name):
         """
```

Comment:
For all templated request handlers, agent name now comes in as a positional parameter
thanks to Tornado machinery.

Reply:
Type hints please.